.flaskenv

```diff
@@ -0,0 +1,2 @@
+FLASK_APP=development/app.py
+FLASK_ENV=development
```

Comment:
You can add the `FLASK_RUN_PORT=5000` back here, since we're using that in the API and need it to be the same. Othwerwise there may be a time when it chooses some other random port.